workshop/4-auth-me-baby.md

```diff
@@ -140,17 +144,17 @@ To use a dropdown in bootstrap we need the bootstrap javascript. To keep things
 Open `/app/index.html`, and below the script tag that brings in `ambitious-arcgis-app.js`, add this license
```

Comment:
remove the step of adding the bootstrap script tag, it's not longer needed now that we're using ember-bootstrap